tests/test_merginreader.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+import pyaro
+
+EBAS_URL = file = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), "testdata", "NILU"
+)
+
+
+def test_stuff():
+    with pyaro.open_timeseries(
+        "mergingreader",
+        datasets=[
+            {"readername": "ascii2netcdf", "filename": EBAS_URL},
```

Comment:
The arguments here are all passed as `args` or `kwargs` to `open_timeseries`. This is the reason I want https://github.com/metno/pyaro/pull/79 implemented. An alternative is to call the "free argument" something, but then the first argument of `MergingReader.__init__()` would be mandated and we don't use it

Reply:
I settled for popping off `filename_or_obj_or_url` and feeding that as the second argument to `open_timeseries`